game/src/devtools/mod.rs

```diff
@@ -74,6 +74,20 @@ impl DevToolsMode {
                     },
                 ])
                 .flex_wrap(ctx, Percent::int(60)),
+                Widget::row(vec![
```

Comment:
I didn't want to crowd the title screen with niche stuff like the OSM viewer and parking mapper, so I hid it here:
![Screenshot from 2021-11-04 14-20-19](https://user-images.githubusercontent.com/1664407/140421726-81a69ccb-f8fc-411a-a92a-72954d34c589.png)
If you say, wanted to go from Santa to OSM viewer, you'd have to load the game exe in the middle to get to this screen. I thought about refactoring this to be in `map_gui`, but got lazy... it'd mean adding a bunch more flags to access each of the devtools.